packages/cli/src/tokens/build/formats/css.ts

```diff
@@ -134,13 +140,50 @@ const colorCategory: Format = {
   },
 };
 
-const calculatedVariable = R.pipe(R.split(/:(.*?);/g), (split) => `${split[0]}: calc(${R.trim(split[1])});`);
+const isDigit = (s: string) => /^\d+$/.test(s);
+const isNumericBorderRadiusToken = (t: TransformedToken) => t.path[0] === 'border-radius' && isDigit(t.path[1]);
+const isNumericOrPrivateSizeToken = (t: TransformedToken) =>
+  t.path[0] === 'size' && (isDigit(t.path[1]) || t.path[1].startsWith('_'));
```

Comment:
I started fooling around with some code to make this "easier" but I'm not sure if it's actually any better. Could maybe be worth it if we did this kind of thing a lot, but maybe not when we only have two such functions 🤔 

```ts
type ElementMatcher = string | ((s: string) => boolean);
const checkArray = R.pipe(
  R.zipWith<ElementMatcher, string, boolean>((matcher, elem) =>
    typeof matcher === 'string' ? elem === matcher : matcher(elem),
  ),
  R.all(R.identity<boolean>),
);

const checkTokenPath = (matchers: ElementMatcher[]) => (t: TransformedToken) => checkArray(matchers, t.path);

const isDigit = (s: string) => /^\d+$/.test(s);
const isNumericBorderRadiusToken = checkTokenPath(['border-radius', isDigit]);
const isNumericOrPrivateSizeToken = checkTokenPath(['size', (x) => isDigit(x) || x.startsWith('_')]);
```

Reply:
hehe, lets save it for later! I am sure there are other stuff that can also be improved.